test/smoke-fort-fails/flang-421353/alloc_struct.f90

```diff
@@ -0,0 +1,14 @@
+module myindexer
+type, public :: indexer
+        character(len=:), allocatable :: escape, ignore_between
+        contains
```

Comment:
**contains** can be removed as we are not defining any procedures here.

Reply:
I agree with your suggested changes, but the more I change it, the less it looks like what was submitted in SWDEV-421353.  If it is OK with you, I would rather keep it as is (since it compiles fine for classic flang and hits the NYI on llvm-flang).   If we were writing the test case I would definitely make the changes you suggested.